.github/workflows/ci-conda-workflow.yml

```diff
@@ -43,33 +52,24 @@ jobs:
           - name: Linux Full Build (Python 3.7)
             os: ubuntu-latest
             python-version: 3.7
-            numpy-version: 1.15
+            numpy-version: 1.19
             install-type: install
             fits: astropy
             test-data: submodule 
             matplotlib-version: 3
             xspec-version: 12.10.1s
 
-          - name: Linux Full Build (Python 3.6)
-            os: ubuntu-latest
-            python-version: 3.6
-            numpy-version: 1.11
-            install-type: install
-            fits: astropy
-            test-data: submodule
-            matplotlib-version: 2
-            xspec-version: 12.10.1s
-
           - name: Linux Build (w/o Astropy or Xspec)
             os: ubuntu-latest
-            python-version: 3.6 
+            python-version: 3.9 
             install-type: install
             test-data: package
-            matplotlib-version: 2
+            matplotlib-version: 3
 
           - name: Linux Build (w/o Matplotlib, Xspec, or test data)
             os: ubuntu-latest
             python-version: 3.7
+            numpy-version: 1.18
```

Comment:
Is the idea here that we are now being "more declarative" in our setup, by explicitly mentioning the required versions of packages?

The only place I could see this being an issue would be if we had a "forward facing" test run where we just want to pick the latest versions available to catch possible problems early - but that's not the place of this particular test.

Reply:
I was using this more as a "minimum support version" test case with some of the other builds being more exploratory

Reply:
This was me mainly thinking out wide, in case anything I was thinking made (or didn't make) sense...